packages/unlock-js/src/__tests__/integration/lock/approveBeneficiary.js

```diff
@@ -0,0 +1,94 @@
+import { itIfErc20 } from '../../helpers/integration'
+import { ethers } from 'hardhat'
+
+let web3Service, chainId, walletService, lock, lockAddress, ERC20
+
+export default ({ isERC20, publicLockVersion }) =>
+  () => {
+    // only v8+
+    if (['v4', 'v6', 'v7'].indexOf(publicLockVersion) === -1) {
```

Comment:
Maybe we do a `itIfVersion` helper? wdyt?

Reply:
yes, had the same idea but wanted to get everything running first